package/obs/rmt-server.spec

```diff
@@ -251,6 +251,10 @@ chrpath -d %{buildroot}%{lib_dir}/vendor/bundle/ruby/*/extensions/*/*/mysql2-*/m
 %dir /var/lib/rmt
 %ghost %{_datadir}/rmt/public/repo
 %ghost %{_datadir}/rmt/public/suma
+
+# The secrets file is created by running the initial rake tasks in the `post` section
+%ghost %{app_dir}/config/secrets.yml.key
```

Comment:
I think you forgot the `secrets.yml.enc` file too

Reply:
absolutely. I even missed copying the right file into the project! I added it.